.github/workflows/deploy-sdk.yml

```diff
@@ -237,3 +237,9 @@ jobs:
           # Help on how to get the webhook URL: https://github.com/marketplace/actions/slack-send#setup-2
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_NOTIFY_RELEASES_WEBHOOK_URL }}
           SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+
+  publish-sample-apps-public-builds:
+    needs: deploy-sonatype
```

Comment:
I rolled out a release and tested it. It wasn't instantly available after Deploy action completed and took around 5–10 minutes to become available, in case that helps.

Reply:
I kept trying to pull it locally on my machine until it became available shortly after deployment action completed. I do agree that syncing across regions can take time, but I still doubt it's instantly available. That said, I'll approve so you can verify your assumptions as well.